CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to snowflake-snowpark-python
 
-Hi, thank you for taking the time to improve Snowflake's Snowpark Python API!
+Hi, thank you for taking the time to improve Snowflake's Snowpark Python or Snowpark pandas APIs!
```

Comment:
```suggestion
Hi, thank you for taking the time to improve Snowflake's Snowpark Python API!
```

The Snowpark pandas API is part of the Snowpark Python API, so I think we should leave this as is.

Reply:
I actually disagree with this bit b/c even though they are part of the same codebase and the pandas API relies on Python API code, it is a different API and I think deserves a callout in the title.

Reply:
Let's get input from someone else, maybe from a snowpark-python codeowner?

Reply:
Actually I just checked with Yijun and Sri offline, this is ok :)

Reply:
@sfc-gh-jdu to be clear, the current version in this PR is ok, or the the current version in main is ok?

Reply:
the current version in this PR